File: omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/comm/exceptions/DiscoveredInvalidPodException.java
package info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.exceptions;

import android.os.ParcelUuid;

import java.util.List;

public class DiscoveredInvalidPodException extends Exception {
    public DiscoveredInvalidPodException(String message, List<ParcelUuid> serviceUUIds) {
        super(message + " service UUIDs: " + serviceUUIds);
    }
}

File: omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/comm/exceptions/DiscoveredInvalidPodException.kt
package info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.exceptions

import android.os.ParcelUuid

class DiscoveredInvalidPodException : Exception {
    constructor(message: String) : super(message)
    constructor(message: String, serviceUUIds: List<ParcelUuid?>) : super("$message service UUIDs: $serviceUUIds")
}